lighthouse-cli/test/cli/run-test.js

```diff
@@ -25,11 +25,18 @@ describe('CLI run', function() {
     const url = 'chrome://version';
     const filename = path.join(process.cwd(), 'run.ts.results.json');
     const flags = getFlags(`--output=json --output-path=${filename} ${url}`);
-    return run.runLighthouse(url, flags, fastConfig).then(_ => {
+    return run.runLighthouse(url, flags, fastConfig).then(passedResults => {
       assert.ok(fs.existsSync(filename));
       const results = JSON.parse(fs.readFileSync(filename, 'utf-8'));
       assert.equal(results.audits.viewport.rawValue, false);
+
+      // passed results match saved results
+      assert.strictEqual(results.generatedTime, passedResults.generatedTime);
+      assert.strictEqual(results.url, passedResults.url);
+      assert.strictEqual(results.audits.viewport.rawValue, passedResults.audits.viewport.rawValue);
+      assert.deepStrictEqual(results.timing, passedResults.timing);
+
       fs.unlinkSync(filename);
     });
-  });
+  }).timeout(60000);
```

Comment:
ah. well...... here was my thinking

60 is currently our timeout.
i can't imagine any mocha `it()` test taking longer than this one in particular.  
so i'm just matching that.  :)

on my machine this test takes 7s to complete, which means on appveyor it could take 45s?
shrug